src/components/shared/SearchCard/SearchCardAction/SearchCardAction.js

```diff
@@ -15,16 +15,21 @@ export const SearchCardAction = ({
   organismNames,
   technology
 }) => {
-  const { getProcessingExperiment } = useOneOffExperiment(accessionCode)
   const pageRendered = usePageRendered()
+  const { datasetAccessions, pollDatasetAccession } = usePollDatasetStatus()
```

Comment:
```suggestion
  const { isProcessing, pollDatasetAccession } = usePollDatasetStatus()
```

Reply:
Now using the new state `isProcessingDataset`, see at a5bf82650a3adc2e4925149acbcb0b3b4820ee21.